Apps/W1/Shopify/app/src/Shipping/Pages/ShpfyShipmentMethodsMapping.Page.al

```diff
@@ -26,6 +26,26 @@ page 30129 "Shpfy Shipment Methods Mapping"
                     ApplicationArea = All;
                     ToolTip = 'Specifies the shipping method in D365BC.';
                 }
+                field("Shipping Charges Type"; Rec."Shipping Charges Type")
+                {
+                    ApplicationArea = All;
+                    ToolTip = 'Specifies the value of the Shipping Charges Type field.';
+                }
+                field("Shipping Charges No."; Rec."Shipping Charges No.")
```

Comment:
The UI could be improved my making this field non editable for the Type::" ". 
As far as I understand the code, then the Type::" " will be used to use the default G/L Account